posthog/api/insight.py

```diff
@@ -326,7 +326,6 @@ def create(self, validated_data: dict, *args: Any, **kwargs: Any) -> Insight:
                     raise serializers.ValidationError("Dashboard not found")
 
                 DashboardTile.objects.create(insight=insight, dashboard=dashboard, last_refresh=now())
-                insight.last_refresh = now()  # set last refresh if the insight is on at least one dashboard
```

Comment:
This has not been used in a long time and not sure it ever actually worked. The actual `last_refresh` is in InsightCachingState at the insight level, and in the cached result packages themselves